### ISSUE TITLE
new lesson: Bubble Sort

### ISSUE THREAD

**Issue by Utkarsh1504** [Utkarsh1504/DSA-Java#19]
### ⭐ Request Type

Add in-depth documentation on the sorting algorithm Bubble Sort
- The markdown file is located [here](https://github.com/Utkarsh1504/DSA-Java/blob/main/lessons/bubblesort.md).

## Lesson Content
The lesson must contain at least these are the topics:
1. Introduction to Sorting
2. Stable and Unstable sorting algorithms
3. Bubble Sort + Why Bubble sort?
4. Complexity Analysis of Bubble sort
    - Time complexity
    - Space complexity
5. Code for Bubble Sort + program explanation

### 💻 Location

This lesson is present under the
Section: `5`
Lesson: `C`
Location Order: `5C`
File-Name: [`bubblesort.md`](https://github.com/Utkarsh1504/DSA-Java/blob/main/lessons/bubblesort.md)

### ❌ Additional Info

You can take the help of other markdown files in the /lesson [directory](https://github.com/Utkarsh1504/DSA-Java/blob/main/lessons/loops.md) so that you get the idea of in which fashion you should write.

- If you don't know how to do markdown, don't worry it is quite easy, just follow this docs-https://guides.github.com/features/mastering-markdown/

- If your concepts are not that good follow this Bubble sort video of @Kunal-Kushwaha (https://www.youtube.com/watch?v=F5MZyqRp_IM)

**Comment by paltannistha** [comment]
I will work on this.

**Comment by Utkarsh1504** [comment]
ok

**Comment by Jisan129** [comment]
Can I work on this ?

**Comment by Utkarsh1504** [comment]
@Jisan129 YES, Just read our contribution guideline and have a look at the website and all the directories inside the repo to get an idea of how this project works. Then you'll be good to go.
